src/Employer.java

```diff
@@ -0,0 +1,61 @@
+// Class Employer
+
+import java.util.ArrayList;
+
+public class Employer {
+
+    public String name; 
+    public int age; 
+    public int salary; 
+    public QueueOfEmployers queueOfEmployers= new QueueOfEmployers();
+    public int amountOfEmployers = 0;
+    
+    Employer(String name, int age, int salary){
+        this.name = name;
+        this.age = age;
+        this.salary = salary;
+    }
+
+    public void addEmployer(String name, int age, int salary){
+        Employer newEmployer = new Employer(name, age, salary);
+        queueOfEmployers.add(newEmployer);
+    }
+
+    public void countEmployers(Employer originalEmployer, Employer currentEmployer, boolean firstCall) {
```

Comment:
as mentioned in class, methods should not include the parameters twice + the instance, even if it's just for the first call